src/worker.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+var queue = require('queue-async');
+var map = require(process.argv[2]);
+
+var q = queue();
+var sources = [];
+
+JSON.parse(process.argv[3]).forEach(function(source) {
+  q.defer(loadSource, source);
+});
+
+function loadSource(source, done) {
+  if (source.mbtiles) sources.push({name: source.name, getTile: require('./mbtiles')(source, done)});
```

Comment:
@morganherlocker for the remote parser, ready is called immediately because there is no initialization step — it is ready to receive requests immediately unlike mbtiles which needs to open a db connection first before getting any tiles to work on.